js/index.js

```diff
@@ -40,3 +40,71 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Selectors
+let cta =  document.getElementById("cta-img");
+let nav = document.querySelectorAll("a");
+let h1 = document.querySelector("h1");
+let button = document.querySelector("button");
+let mcimage = document.getElementById("middle-img")
+let h4 = document.querySelectorAll("h4");
+let txtcontent = document.querySelectorAll("p");
+
+
+
+
+
+//Updates
+cta.setAttribute('src', siteContent["cta"]["img-src"])
+mcimage.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+let navArray = Array.from(nav); 
+navArray[0].textContent = siteContent["nav"]["nav-item-1"]; 
+navArray[1].textContent = siteContent["nav"]["nav-item-2"]; 
+navArray[2].textContent = siteContent["nav"]["nav-item-3"]; 
+navArray[3].textContent = siteContent["nav"]["nav-item-4"]; 
+navArray[4].textContent = siteContent["nav"]["nav-item-5"]; 
+navArray[5].textContent = siteContent["nav"]["nav-item-6"]; 
+
+h1.textContent = siteContent["cta"]["h1"];
+h1.style.width = "41%";
+button.textContent = siteContent["cta"]["button"];
+
+let h4Array = Array.from(h4);
+h4[0].textContent = siteContent["main-content"]["features-h4"]
+h4[1].textContent = siteContent["main-content"]["about-h4"];
+h4[2].textContent = siteContent["main-content"]["services-h4"];
+h4[3].textContent = siteContent["main-content"]["product-h4"];
+h4[4].textContent = siteContent["main-content"]["vision-h4"];
+h4[5].textContent =siteContent["contact"]["contact-h4"];
+
+
+let txtArray = Array.from(txtcontent);
+txtArray[0].textContent = siteContent["main-content"]["features-content"];
+txtArray[1].textContent =siteContent["main-content"]["about-content"];
+txtArray[2].textContent = siteContent["main-content"]["services-content"];
+txtArray[3].textContent = siteContent["main-content"]["product-content"];
+txtArray[4].textContent = siteContent["main-content"]["vision-content"];
+txtArray[5].textContent = siteContent["contact"]["address"];
+txtArray[6].textContent = siteContent["contact"]["phone"];
+txtArray[7].textContent = siteContent["contact"]["email"];
+txtArray[8].textContent = siteContent["footer"]["copyright"];
+
+//Color change
+navArray.forEach(function(navitem){
```

Comment:
I'd recommend practicing ES6 here. It's okay though because I know you're still learning. ```navArray.forEach((navitem) => {
navitem.style.color = 'green'
})```